survey-runner.tf

```diff
@@ -110,10 +110,20 @@ module "survey-runner-vpc" {
   vpc_cidr_block = "${var.vpc_cidr_block}"
 }
 
+module "ebs_bckup" {
```

Comment:
Also, i think we should move this entire module into the `survey-runner-queue` module.